blocks/quilted-image-block/features/quilted-image/default.jsx

```diff
@@ -0,0 +1,254 @@
+			<HeadingSection>
+				{headline ? <Heading>{headline}</Heading> : null}
+				<div className={`${BLOCK_CLASS_NAME}--wrapper`}>
+					<Link
+						href={item1Action || "#"}
+						className={`${BLOCK_CLASS_NAME}--media-panel  ${
+							fullWidthImage === "top" ? `${BLOCK_CLASS_NAME}--wrapper--top` : ""
+						}`}
+					>
+						<Image src={image1URL} style={{ aspectRatio: image1AspectRatio }} />
+						<Stack className={`${BLOCK_CLASS_NAME}--overlay`} inline>
+							{overlay1Text ? (
+								<Paragraph className={`${BLOCK_CLASS_NAME}--overlay-text-${overlay1TextVariant}`}>
+									{overlay1Text}
+								</Paragraph>
+							) : null}
+							{button1Text ? (
+								<Button variant={button1Variant} size="small" assistiveHidden>
+									{button1Text}
+								</Button>
+							) : null}
+						</Stack>
+					</Link>
+					<Link href={item2Action || "#"} className={`${BLOCK_CLASS_NAME}--media-panel`}>
+						<Image src={image2URL} style={{ aspectRatio: image2AspectRatio }} />
+						<Stack className={`${BLOCK_CLASS_NAME}--overlay`} inline>
+							{overlay2Text ? (
+								<Paragraph className={`${BLOCK_CLASS_NAME}--overlay-text-${overlay2TextVariant}`}>
+									{overlay2Text}
+								</Paragraph>
+							) : null}
+							{button2Text ? (
+								<Button variant={button2Variant} size="small" assistiveHidden>
+									{button2Text}
+								</Button>
+							) : null}
+						</Stack>
+					</Link>
+					<Link
+						href={item3Action || "#"}
+						className={`${BLOCK_CLASS_NAME}--media-panel ${
+							fullWidthImage === "bottom" ? `${BLOCK_CLASS_NAME}--wrapper--bottom` : ""
+						}`}
+					>
+						<Image src={image3URL} style={{ aspectRatio: image3AspectRatio }} />
+						<Stack className={`${BLOCK_CLASS_NAME}--overlay`} inline>
+							{overlay3Text ? (
+								<Paragraph className={`${BLOCK_CLASS_NAME}--overlay-text-${overlay3TextVariant}`}>
+									{overlay3Text}
+								</Paragraph>
+							) : null}
+							{button3Text ? (
+								<Button variant={button3Variant} size="small" assistiveHidden>
+									{button3Text}
+								</Button>
+							) : null}
+						</Stack>
+					</Link>
+				</div>
+			</HeadingSection>
+		</div>
+	);
+}
+
+QuiltedImage.label = "Quilted Image - Arc Block";
+
+QuiltedImage.icon = "picture-polaroid-album-1";
+
+QuiltedImage.propTypes = {
+	customFields: PropTypes.shape({
+		headline: PropTypes.string.tag({
+			label: "Headline",
+			defaultValue: "",
+			description: "Headline text describing all images",
+		}),
+		fullWidthImage: PropTypes.oneOf(["top", "bottom"]).tag({
+			label: "Full Width Image Location",
+			defaultValue: "bottom",
+		}),
+		image1URL: PropTypes.string.tag({
+			label: "Image URL",
+			searchable: "image",
+			description: "URL of first image, displayed at 4:3 aspect ratio.",
+			group: "Image 1",
+		}).isRequired,
+		image1AspectRatio: PropTypes.oneOf(["16/9", "4/3"]).tag({
+			labels: {
+				"16/9": "16:9",
+				"4/3": "4:3",
+			},
```

Comment:
Small recommendation, non-blocking...

```
const ASPECT_RATIO_MAP = {
	"16/9": "16:9",
	"4/3": "4:3",
};

/* ... */

		image1AspectRatio: PropTypes.oneOf(Object.keys(ASPECT_RATIO_MAP)).tag({
			labels: ASPECT_RATIO_MAP,
```

Then apply to all images listed so that if we ever add more aspect ratios, it would apply to all.

Reply:
possibly apply this to the `['dark', 'light']` and `['primary', 'secondary']` as well

Reply:
I'm now wondering if the `Button` variants should be exposed (`Button.variants`?) so that anything that uses the button could expose the options for less maintenance for anything that could use `Button`.

Reply:
> I'm now wondering if the `Button` variants should be exposed (`Button.variants`?) so that anything that uses the button could expose the options for less maintenance for anything that could use `Button`.

❤️

Reply:
@nschubach That's a great idea - we should create a ticket to explore this